fe/fe-core/src/main/java/org/apache/doris/nereids/trees/expressions/functions/executable/NumericArithmetic.java

```diff
@@ -132,7 +132,7 @@ public static Expression addLargeIntLargeInt(LargeIntLiteral first, LargeIntLite
     @ExecFunction(name = "add")
     public static Expression addDoubleDouble(DoubleLiteral first, DoubleLiteral second) {
         double result = first.getValue() + second.getValue();
-        return checkOutputBoundary(new DoubleLiteral(result));
+        return new DoubleLiteral(result);
```

Comment:
all of these function has test case for return inf and Nan?

Reply:
Not all of them, I'll add some cases.